catalog/Dockerfile

```diff
@@ -18,14 +19,16 @@ EXPOSE 3000
 RUN mkdir /opt/app
```

Comment:
Remove these, too - and all of the install code above besides nginx and gettext.

Reply:
Might be a good idea to switch to a newer ubuntu, too - 16.04 is pretty ancient at this point. (Or use the nginx image?)